.github/workflows/release-to-pypi.yml

```diff
@@ -0,0 +1,80 @@
+name: Release to PyPI
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build-wheels:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - name: Check out the release commit
+      uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: arm64
+      if: runner.os == 'Linux'
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install Python packages needed for wheel build and upload
+      run: |
+        python -m pip install --upgrade pip setuptools
```

Comment:
Nitpick: This step shouldn't be needed: if we have a pyproject.toml, we don't need `setuptools`, and `pip` is automatically updated to the latest version by the setup-python action.